webapp/channels/src/components/three_days_left_trial_modal/three_days_left_trial_modal.tsx

```diff
@@ -102,10 +96,10 @@ function ThreeDaysLeftTrialModal(props: Props): JSX.Element | null {
         },
     ], []);
 
-    let headerText = formatMessage({id: 'three_days_left_trial.modal.title', defaultMessage: 'Your trial ends soon'});
-    let headerSubtitleText = formatMessage({id: 'three_days_left_trial.modal.subtitle', defaultMessage: 'There is still time to explore what our paid plans can help you accomplish.'});
+    const headerText = formatMessage({id: 'three_days_left_trial.modal.title', defaultMessage: 'Your trial ends soon'});
+    const headerSubtitleText = formatMessage({id: 'three_days_left_trial.modal.subtitle', defaultMessage: 'There is still time to explore what our paid plans can help you accomplish.'});
```

Comment:
Since these are const now, we may consider putting them nearer to where they are actually used (we could even drop the variable, if they are used in only one place).